frontend/post/postDetailsDirective.js

```diff
@@ -615,20 +619,19 @@
         };
 
         commentCtrl.canReply = function canReply() {
-            return commentCtrl.post.state === 'deleted';
+            return commentCtrl.showReplies && !commentCtrl.isDeletedPost() && !commentCtrl.isInstInactive();
```

Comment:
This change should break some tests. If none is broken or we don't have tests to cover that function or those that exist are wrong.

Reply:
there was no test for the method